MdePkg/Include/UniversalPayload/SmBiosTableHob.h

```diff
@@ -0,0 +1,26 @@
+/** @file

```

Comment:
This file name is not consistent to the root bridge one. that one has the prefix "Pld" while this doesn't.
Do you think we need to add prefix "Pld" or remove the prefix?

Reply:
I can add "Pld"